tests/plugins/test_fetch_maven_artifacts.py

```diff
@@ -500,12 +543,22 @@ def test_fetch_maven_artifacts_nvr_bad_checksum(tmpdir, docker_tasker):
 
 
 @responses.activate  # noqa
-def test_fetch_maven_artifacts_nvr_bad_url(tmpdir, docker_tasker):
+def test_fetch_maven_artifacts_nvr_bad_url(tmpdir, docker_tasker, reactor_config_map):
     """Err on download errors for artifact from Koji build."""
     workflow = mock_workflow(tmpdir)
     mock_koji_session()
     mock_fetch_artifacts_by_nvr(str(tmpdir))
     mock_nvr_downloads(overrides={1269850: {'status': 404}})
+
+    if reactor_config_map:
```

Comment:
should call make_and_store_reactor_config_map

Reply:
yeah forgot that one :)